includes/class-cron.php

```
@@ -130,6 +137,57 @@ public function ccss_regeneration_cron() {
+			if ( ! empty( $audit['high_value_urls'] ) && is_array( $audit['high_value_urls'] ) ) {
+				foreach ( $audit['high_value_urls'] as $high_url ) {
+					$clean = esc_url_raw( (string) $high_url );
+					if ( ! empty( $clean ) && ! in_array( $clean, $urls, true ) ) {
+						$urls[] = $clean;
+					}
+				}
+			}
+
+			foreach ( $urls as $scan_url ) {
+				foreach ( array( 'mobile', 'desktop' ) as $scan_strategy ) {
+					Pagespeed::queue_scan( $scan_url, $scan_strategy );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'PerformanceOptimise\Inc\Pagespeed' in method 'web_vitals_rescan_cron'. (undefined)

(StaticAccess)